defaults/main.yml

```diff
@@ -23,6 +23,10 @@ storage_pool_defaults:
   raid_chunk_size: null
   raid_metadata_version: null
 
+  compression: null
+  deduplication: null
+  vdo_logical_size: null
```

Comment:
This needs updating to `vdo_pool_size`.

Reply:
I missed this by greping only over .py files. Fixed.